draft-wang-ppm-differential-privacy.md

```diff
@@ -239,11 +261,135 @@ The following considerations are out-of-scope for this document:
 
 This document uses the same conventions for error handling as {{!DAP}}.
 
-> TODO: add more
+Let `exp(EPSILON)` denote raising the numeric constant `e` to the power of
+`EPSILON`.
+
+# Security Goals and Trust Model {#overview}
+
+## Differential Privacy
+
+DP formalizes a property of any randomized algorithm that takes in a sequence
+of measurements, aggregates them, and outputs an aggregate result. Intuitively,
+this property guarantees that no single measurement signficantly impacts the
+value of the aggregate result. This intuition is formalized by {{DMNS06}} as
+follows.
+
+> CP: The following is might be too jargony for an informative RFC, but for now
+> I think we're all just trying to agree on the definition. Once we have
+> consensus among ourselves, we can punt this to the appendix and leave a less
+> formal description here.
+
+We say that two batches of measurements `D1` and `D2` are "neighboring" if they
+are the same length and contain all the same measurements except one (i.e., the
+symmetric difference between the multisets contains two elements).
+
+Let `p(S, D, R)` denote the probability that randomized algorithm `S`, on input
+of measurements `D`, outputs aggregate result `R`.
```

Comment:
The rest of the document seems to call the randomized algorithm as `R`, and aggregate result as `S`.

Reply:
also one nit thing: I don't know if you intend to use a mixture of "randomized algorithm" and "DP mechanism", but they seem to mean the same to me.

Reply:
* Arg... I meant to use `S` for the algorithm and `r` for the aggregate result. Fixed now (hopefully).
* They're slightly different because the "DP mechanism" just generates noise; the "randomized algorithm" here aggregates then invokes a DP mechanism.